File: app/src/main/java/com/evaluation/model/asset/Asset.java
package com.evaluation.model.asset;

import com.google.gson.annotations.SerializedName;

public class Asset {

    @SerializedName("id")
    private int id;

    @SerializedName("title")
    private String title;

    @SerializedName("overview")
    private String overview;

    @SerializedName("backdrop_path")
    private String posterPath;

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return overview;
    }

    public String getPosterPath() {
        return posterPath;
    }
}

File: app/src/main/java/com/evaluation/model/asset/Asset.kt
package com.evaluation.model.asset

import com.google.gson.annotations.SerializedName

/**
 * @author Vladyslav Havrylenko
 * @since 10.03.2020
 */
data class Asset(
    @SerializedName("id")
    val id: Int,

    @SerializedName("title")
    val title: String,

    @SerializedName("overview")
    val overview: String,

    @SerializedName("backdrop_path")
    val posterPath: String
)